ckan/templates/package/activity.html

```diff
@@ -8,5 +8,51 @@ <h1 class="hide-heading">
       {{ _('Activity Stream') }}
     {% endblock %}
   </h1>
-  {% snippet 'snippets/activity_stream.html', activity_stream=activity_stream, id=id, object_type='package' %}
+
+  {# allow extensions to continue using this template without new vars #}
+  {% if activity_types is defined %}
+  <div 
+    id="activity_types_filter"
+    style="margin-bottom: 15px;"
+    data-module="activity-stream"
+    >
+    <label for="activity_types_filter_select">Activity type</label>
+    <select id="activity_types_filter_select">
```

Comment:
We really should not show "package" to the end user, specially if other activities have dataset in it. And they are not translatable.
So maybe instead of using the keys with `.title()` we need to somehow use the new [humanize](https://github.com/ckan/ckan/blob/c00891b0a35c7c334701bbcb56cfad61dbfedabe/ckan/lib/helpers.py#L962) functions to get a proper label

![Screenshot from 2022-05-03 12-49-02](https://user-images.githubusercontent.com/200230/166440820-056a2bb2-165b-4ede-a63d-54f2d5039ff4.png)

Reply:
This is a simple proposal to humanize all activities: https://github.com/avdata99/ckan/commit/7a55c0e9a564da110dec2092f24a05bc4b77a2c1
The ´humanize_entity_type´ function doesn't fit exactly in what we need but we can use it @amercader